Parser/pgen/keywordgen.py

```diff
@@ -0,0 +1,61 @@
+"""Generate Lib/keyword.py from the Grammar and Tokens files using pgen"""
+
+import argparse
+
+from .pgen import ParserGenerator
+
+TEMPLATE = r'''\
```

Comment:
The trailing \ is wrong: because of that,  "#! /usr/bin/env python3" isn't the first line of keyword.py, *but* it is a valid Python file :-)